src/jsdocUtils.js

```diff
@@ -147,95 +175,68 @@ const namepathDefiningTags = [
   'namespace',
 ];
 
-const namepathPointingTags = [
-  // NOT USEFUL WITHOUT NAMEPATH
+const tagsWithOptionalNamepath = [
+  ...namepathDefiningTags,
```

Comment:
As per my comment for `tagsWithOptionalType`, I would prefer that `tagsWithOptionalNamepath` does not include `namepathDefiningTags` but instead lists the non-mandatory items only, and that `tagMightHaveNamepath` checks both `tagMightHaveNamepath` and `tagsWithMandatoryNamepath`.

Reply:
A bit more complicated here because `namepathDefiningTags` is also used for other utility as well. And is happened, all of the mandatory namepath tags are namepath defining. I left those in both lists and made `tagMightHaveNamepath` check all three in case they change in the future.